pyproject.toml

```diff
@@ -19,18 +19,18 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-jsonschema = "^3.2.0"
+python = "^3.7"
+jsonschema = "^4.4.0"
 strict_rfc3339 = "^0.7"
-werkzeug = "0.16.1"
+werkzeug = "^2.0.0"
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.4.1"
+pytest = "^7.1.0"
```

Comment:
This is to avoid [this problem](https://stackoverflow.com/questions/69528842/an-error-while-trying-to-execute-tests-on-python-3-10-with-pytest)